apps/ejabberd/src/mod_muc.erl

```diff
@@ -1127,3 +1132,52 @@ can_access_room(_, From, To) ->
         {ok, CanAccess} -> CanAccess
     end.
 
+online_rooms_number() ->
+    lists:sum([online_rooms_number(Host) || Host <- ?MYHOSTS]).
+
+online_rooms_number(Host) ->
+    try
+        Supervisor = gen_mod:get_module_proc(Host, ejabberd_mod_muc_sup),
+        Stats = supervisor:count_children(Supervisor),
+        proplists:get_value(active, Stats)
+    catch _:_ ->
+              0
+    end.
+
+hibernated_rooms_number() ->
+    lists:sum([hibernated_rooms_number(Host) || Host <- ?MYHOSTS]).
+
+hibernated_rooms_number(Host) ->
+    try
+        count_hibernated_rooms(Host)
+    catch _:_ ->
+              0
```

Comment:
In this particular case I just followed vim's indentation which is in most cases what we want.